test/functionalTest/cases/2933_PATCH_offending_attributes/PATCH_offending_attributes.test

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PATCH /v2/entities/ID/attrs fails must give offending attributes
+
+
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+
+--SHELL--
+
+#
+# 01. POST /v2/entities Entity creation
+# 02. PATCH /v2/entities/Room1/attrs, to update the value of attributes
+# 03. GET  /v2/entities
+# 04. DELETE /v2/entities
+# 05. GET /v2/entities
+#
+
+echo '01. POST /v2/entities, to create Room1 with temperature and pressure.'
+echo '====================================================================='
+payload='{
+  "temperature": {
+    "value": 23,
+    "type": "Float"
+  },
+  "pressure": {
+    "value": 720,
+    "type": "Float"
+  }
+}'
+orionCurl --url /v2/entities/Room1/attrs?options=append --payload "$payload"
+echo
+echo
+
+
+echo '02. PATCH /v2/entities/Room1/attrs, to update the value of pressure attribute.'
+echo '=============================================================================='
+payload='{
+  "temperature": {
+    "value": 24,
+    "type": "Float"
+  },
+  "pressur": {
+    "value": 730,
+    "type": "Float"
+  }
+}'
+orionCurl --url /v2/entities/Room1/attrs?options=append --payload "$payload" -X PATCH
+echo
+echo
+
+echo '03. GET Entities.'
+echo '================='
+orionCurl --url /v2/entities
+echo
+echo
+
+echo '04. DELETE Entities.'
+echo '=================='
+orionCurl --url /v2/entities/Room1 -X DELETE
+echo
+echo
+
+echo '05. GET Entities.'
+echo '================='
+orionCurl --url /v2/entities
+echo
+echo
```

Comment:
Probably this steps are not needed... or maybe I'm missing something?

Reply:
On a second thought, maybe 03 is a good idea, to check the status of the entity after the update attemps. But 04 and 05 should be removed.